rust/cardano-chain-follower/src/follow.rs

```diff
@@ -74,9 +73,9 @@ impl ChainFollower {
         ChainFollower {
             chain,
             end,
-            previous: UNKNOWN_POINT,
+            previous: Point::UNKNOWN,
             current: start,
-            fork: 1, // This is correct, because Mithril is Fork 0.
+            fork: 1.into(), // This is correct, because Mithril is Fork 0.
```

Comment:
Magic number, better to make it as a `const` of `Fork` type, the same as `Fork::IMMUTABLE`

Reply:
I was not sure, is this consider a backfill? @stevenj